presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -60,45 +68,41 @@ protected QueryRunner createQueryRunner()
     @Test
     public void testCteExecutionWhereOneCteRemovedBySimplifyEmptyInputRule()
     {
-        String sql = "WITH t as(select orderkey, count(*) as count from (select orderkey from orders where false) group by orderkey)," +
+        @Language("SQL") String sql = "WITH t as(select orderkey, count(*) as count from (select orderkey from orders where false) group by orderkey)," +
                 "t1 as (SELECT * FROM orders)," +
                 " b AS ((SELECT orderkey FROM t) UNION (SELECT orderkey FROM t1)) " +
                 "SELECT * FROM b";
         QueryRunner queryRunner = getQueryRunner();
-        compareResults(queryRunner.execute(getMaterializedSession(),
-                        sql),
-                queryRunner.execute(getSession(),
-                        sql));
+        CTEInfoValue t = new CTEInfoValue("t", 1, false);
```

Comment:
can we have less abbreviated variable names?

Reply:
I've moved the CTEInfoValue objects creation inline with the ImmutableList similar to the paradigm in other tests which circumvents the need to define variable names.